packages/open-next/src/core/require-hooks.ts

```diff
@@ -144,6 +142,7 @@ export function applyOverride() {
     const hookResolved = isApp()
       ? requestMapApp.get(request)
       : requestMapPage.get(request);
+    //! biome-ignore lint/style/noParameterAssign: <explanation> <-- doesn't work
```

Comment:
It seems there's an issue with Biome. Here’s a reproduction link: [Biome Playground](https://biomejs.dev/playground/?code=LwAvACAAYgBpAG8AbQBlAC0AaQBnAG4AbwByAGUAIABsAGkAbgB0AC8AYwBvAG0AcABsAGUAeABpAHQAeQAvAHUAcwBlAEEAcgByAG8AdwBGAHUAbgBjAHQAaQBvAG4AOgAgAFQAaABpAHMAIABjAG8AbQBtAGUAbgB0ACAAZABvAGUAcwAgAG4AbwB0ACAAdwBvAHIAawAgAHcAaQB0AGgAIABhAG4AIABpAG4AbgBlAHIAIABjAG8AbQBtAGUAbgB0ACAAaQBuACAAdABoAGUAIABmAHUAbgBjAHQAaQBvAG4ACgBjAG8AbgBzAHQAIABmAG8AbwAwACAAPQAgAGYAdQBuAGMAdABpAG8AbgAgACgAYgBhAHIAOgAgAHMAdAByAGkAbgBnACkAIAB7AAoACQAvAC8AIABiAGkAbwBtAGUALQBpAGcAbgBvAHIAZQAgAGwAaQBuAHQALwBzAHQAeQBsAGUALwBuAG8AUABhAHIAYQBtAGUAdABlAHIAQQBzAHMAaQBnAG4AOgAgAFQAaABpAHMAIABjAG8AbQBtAGUAbgB0ACAAZABvAGUAcwAgAG4AbwB0ACAAdwBvAHIAawAKAAkAYgBhAHIAIAA9ACAAIgBiAGEAegAiADsACgB9AC4AYgBpAG4AZAAoAG4AdQBsAGwALAAgACIAYgBhAHIAIgApADsACgAKAGMAbwBuAHMAdAAgAGYAbwBvADEAIAA9ACAAZgB1AG4AYwB0AGkAbwBuACAAKABiAGEAcgA6ACAAcwB0AHIAaQBuAGcAKQAgAHsACgAJAC8ALwAgAGIAaQBvAG0AZQAtAGkAZwBuAG8AcgBlACAAbABpAG4AdAAvAHMAdAB5AGwAZQAvAG4AbwBQAGEAcgBhAG0AZQB0AGUAcgBBAHMAcwBpAGcAbgA6ACAAVABoAGkAcwAgAGMAbwBtAG0AZQBuAHQAIABkAG8AZQBzACAAbgBvAHQAIAB3AG8AcgBrAAoACQBiAGEAcgAgAD0AIAAiAGIAYQB6ACIAOwAKAH0ALgBiAGkAbgBkACgAbgB1AGwAbAAsACAAIgBiAGEAcgAiACkAOwAKAAoALwAvACAAYgBpAG8AbQBlAC0AaQBnAG4AbwByAGUAIABsAGkAbgB0AC8AYwBvAG0AcABsAGUAeABpAHQAeQAvAHUAcwBlAEEAcgByAG8AdwBGAHUAbgBjAHQAaQBvAG4AOgAgAFQAaABpAHMAIABjAG8AbQBtAGUAbgB0ACAAdwBvAHIAawBzACAAaQBmACAAdABoAGUAcgBlACAAaQBzACAAbgBvACAAaQBuAG4AZQByACAAYwBvAG0AbQBlAG4AdAAgAGkAbgAgAHQAaABlACAAZgB1AG4AYwB0AGkAbwBuAAoAYwBvAG4AcwB0ACAAZgBvAG8AMgAgAD0AIABmAHUAbgBjAHQAaQBvAG4AIAAoAGIAYQByADoAIABzAHQAcgBpAG4AZwApACAAewAKAAkAYgBhAHIAIAA9ACAAIgBiAGEAegAiADsACgB9AC4AYgBpAG4AZAAoAG4AdQBsAGwALAAgACIAYgBhAHIAIgApADsACgA%3D)

Reply:
Just set it to `// biome-ignore lint: Should be lint/style/noParameterAssign, but there is a bug in biome`

Reply:
+1.

And extra +1 if there is a link to the biome bug

Reply:
The issue was confirmed: https://github.com/biomejs/biome/issues/4519